### PR TITLE
Mesh copy with test

### PR DESCRIPTION
**Pull request by kerautret**

### CODE REVIEW

**Comment by dcoeurjo** [comment]
Sounds ok... Could you please add a changelog entry?

**Comment by dcoeurjo** [comment]
BTW : when adding a QuadFAce to a mesh (or polygon), what is the convention (CW of CCW) to have the normal vectors ok in OBJ viewers ?
The Doc is a bit unclear on that.

**Comment by kerautret** [comment]
For the order it is CCW at least for OBJ, I add it to a note to the respecting method thank

**Comment by kerautret** [comment]
@dcoeurjo done.

**Comment by dcoeurjo** [comment]
ok thanks. merging

File: ChangeLog.md [hunk 1]
```diff
@@ -54,6 +54,11 @@
    [#937](https://github.com/DGtal-team/DGtal/pull/937))
  - New methods to generate basic 3D tubular meshes and height fields. New mesh module documentation added.
    (Bertrand Kerautret, [#969](https://github.com/DGtal-team/DGtal/pull/969))
+ - Add of copy constructor and copy operator on Mesh object (and doc adds about vertex ordering for obj format).
```
Comment (dcoeurjo) [inline ChangeLog.md]: doc adds -> documentation added

Comment (dcoeurjo) [inline ChangeLog.md]: Add of -> New ...

**Comment by dcoeurjo** [comment]
(sorry there are still minor typo in the changelog)

**Comment by kerautret** [comment]
Thanks that done;)

**Comment by kerautret** [comment]
can we merge ?

**Comment by dcoeurjo** [comment]
Yep.. doing it

**Pull request merged**